root/genre/CreateGenre.js

```diff
@@ -21,9 +21,9 @@ type Props = {
 const CreateGenre = ({
   form,
 }: Props): React$Element<typeof Layout> => (
-  <Layout fullWidth title={l('Add a new genre')}>
+  <Layout fullWidth title={'Add a new genre'}>
```

Comment:
How come a lot of strings in this PR aren't using `l_admin`? Not sure I understand when it should be used and when it shouldn't.

Reply:
It should be used in files having a mix of translated and not translated strings. It seems fine to not use it here as genre editors are privileged users.